packages/react-app/src/App.css

```diff
@@ -1,4 +1,8 @@
 .App {
   text-align: center;
   padding-left: calc(100vw - 100%);
+}
+
+.sr-only {
+  display: none;
```

Comment:
Apparently this is considered bad practice for accessibility https://stackoverflow.com/questions/19758598/what-is-sr-only-in-bootstrap-3

Reply:
This was suggested by the `react-leafflet-draw` library docs, but apparently (at least in my browser) it's not necessary. Removing.